test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_UI_SetGlobalProperties_WARNINGS.lua

```diff
@@ -0,0 +1,161 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one HMI-portions
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this test case when UI.SetGlobalProperties gets WARNINGS is checked
+-- 1. Used preconditions: App is activated and registered SUCESSFULLY
+-- 2. Performed steps: 
+-- MOB -> SDL: sends SetGlobalProperties
+-- SDL -> HMI: resends UI.SetGlobalProperties with imageType if vrHelp which is not supported by HMI, and TTS.SetGlobalProperties with all valid params
+-- HMI -> SDL: VR.SetGlobalProperties (WARNINGS), TTS.SetGlobalProperties (SUCCESS)
+--
+-- Expected result:
+-- SDL -> MOB: appID: (WARNINGS, success: true: SetGlobalProperties)
```

Comment:
@vvvakulenko, please move SetGlobalProperties out of brackets, appID is not a parameter of SetGlobalProperties.
As example:
SDL -> MOB: SetGlobalProperties (resultcode: WARNINGS, success: true)